src/main/sensors/voltage.c

```diff
@@ -218,7 +226,8 @@ void voltageMeterESCInit(void)
 {
 #ifdef USE_ESC_SENSOR
     memset(&voltageMeterESCState, 0, sizeof(voltageMeterESCState_t));
-    pt1FilterInit(&voltageMeterESCState.filter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatLpfPeriod), HZ_TO_INTERVAL(50)));
+    pt1FilterInit(&voltageMeterESCState.filter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatDisplayLpfPeriod), HZ_TO_INTERVAL(50)));
+    pt1FilterInit(&voltageMeterESCState.sagFilter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatSagLpfPeriod), HZ_TO_INTERVAL(200)));
```

Comment:
Should be 50hz.

Reply:
If you could let me know how to correctly get it to run at 200hz, I'd be grateful.  At the fastest I only have a 100ms period for the lowpass applied to battery, or 10hz.  I need the sampling of the input to be faster than 50hz or the output is very jittery.

Reply:
I think if the battery is sampled at 200, as I did it, then all those hz to intervals should be 200?